src/util/misc/process-helper.ts

```diff
@@ -1,9 +1,17 @@
 import * as child_process from "child_process";
 import { out } from "../interaction";
 
-export function execAndWait(command: string): Promise<number> {
+export function execAndWait(command: string, onStdOut?: (text: string) => void, onStdErr?: (text: string) => void): Promise<number> {
   return new Promise((resolve, reject) => {
+    if (!onStdOut) {
+      onStdOut = text => out.text(text);
+    }
+    if (!onStdErr) {
+      onStdErr = text => out.text(text);
```

Comment:
Should we add an `out.warn` or `out.err` or something like that? With the idea that using it would output to stderr instead of stdout?